terraform.tfvars.example

```diff
@@ -198,6 +198,8 @@ node_pools = {
   #  memory           = 32,
   #  node_pool_size   = 1,
   #  boot_volume_size = 150,
+  #  eviction_grace_duration = 2,
```

Comment:
Different value for `eviction_grace_duration` between tfvars example (2), lookup default (0), and service default (60m)